be/src/agent/task_worker_pool.cpp

```diff
@@ -1259,11 +1217,6 @@ void TaskWorkerPool::_report_tablet_worker_thread_callback() {
             DorisMetrics::instance()->report_all_tablets_requests_skip->increment(1);
             continue;
         }
-        if (!build_all_report_tablets_info_status.ok()) {
-            LOG(WARNING) << "build all report tablets info failed. status: "
-                         << build_all_report_tablets_info_status;
```

Comment:
If build failed, some logic in handle report maybe wrong, it think this logic should not be removed.

Reply:
In fact, `build_all_report_tablets_info` always returns OK, I even consider let `build_all_report_tablets_info` return void.